__tests__/app.test.js

```diff
@@ -9,12 +9,28 @@ beforeEach(() => seed(data));
 afterAll(() => db.end());
 
 describe("GET /api", () => {
-  test.only("200: Responds with an object detailing the documentation for each endpoint", () => {
+  test("200: Responds with an object detailing the documentation for each endpoint", () => {
     return request(app)
       .get("/api")
       .expect(200)
       .then(({ body: { endpoints } }) => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
-});
\ No newline at end of file
+});
+
+describe.only("GET /api/topics", () => {
+  test("status 200 - responds with the requested array containing all of the topics", () => {
+    return request(app)
+    .get("/api/topics")
+    .expect(200)
+    .then(({body}) => {
+      expect(body.topics).toHaveLength(3);
+      body.topics.forEach((topic) => {
+        expect(typeof topic.description).toBe("string");
+        expect(typeof topic.slug).toBe("string");
+        expect(topic).not.toHaveProperty("img_url");
```

Comment:
We don't need this last expect statement here. As long as we're showing the description and slug is there, you're all good 👍